### ISSUE TITLE
`ob deploy push` fails on systemd-timesyncd.service

### ISSUE THREAD

**Issue by mightybyte** [obsidiansystems/obelisk#670]
I just upgraded an app to the latest obelisk and am now getting this failure when I run `ob deploy push`:

```
Mar 16 16:16:08 server.example.com systemd[26562]: systemd-timesyncd.service: Failed at step STATE_DIRECTORY spawning /nix/store/f1lpfdlzqa5l711zmkg32zm6nrxp95m6-systemd-243/lib/systemd/systemd-timesyncd: Not a directory
Mar 16 16:16:08 server.example.com systemd[1]: systemd-timesyncd.service: Main process exited, code=exited, status=238/STATE_DIRECTORY
Mar 16 16:16:08 server.example.com systemd[1]: systemd-timesyncd.service: Failed with result 'exit-code'.
Mar 16 16:16:08 server.example.com systemd[1]: Failed to start Network Time Synchronization.
```

The app is currently using obelisk commit `3f0d2826b349dbda16cd3d28fbdba4d5b9874e3f`.

**Comment by 3noch** [comment]
Strange....I just deployed with develop a few days ago without hiccup.. I'll try on that commit.

**Comment by mightybyte** [comment]
I wonder if it might be related to which version I'm coming from.  I believe my previously deployed version was using obelisk commit `66a78621da05d6ded1cd849466de7623d94ee2ca`.

**Comment by 3noch** [comment]
That's good info. I'll try that transition as well if I can't repro initially.

**Comment by alexfmpe** [comment]
Is this failure happening on a NixOS?

**Comment by mightybyte** [comment]
Yes.

**Comment by bidigo** [comment]
We had a similar issue a while ago and "fixed" it by manually deleting `/var/lib/systemd/timesync` and `/varlib/private` directories and restarting the `timesyncd.service`. Since then our ob deploys are working fine.

**Comment by mightybyte** [comment]
@bidigo Thanks, thank did it!  All I had to do was delete `/var/lib/systemd/timesync` and `/var/lib/private` on the deploy machine.  Then `ob deploy push` from the build machine worked.  (I tried to restart `timesyncd.service` as well but it said `Unit timesyncd.service not found.`)

@3noch I'm fine closing this issue.  Leaving it open in case you want to track this for some other kind of internal fix.
